www/site/content/docs/reference/config.md

```diff
@@ -379,3 +379,26 @@ Default
 
 Used by
 : `chef-run`
+
+### Features
+
+Enable and disable experimental features for Chef Workstation.
+
+#### Example
+
+```toml
+[features]
+analyze = true
+```
+
+Description
+: List of experimental features that can be turn on and off. In the above example, there is one turned on feature, the `analyze` feature. Every feature flag has a way to be turned on through the command line via environment variables. The respective environment variable for the `analize` feature would be `CHEF_FEAT_ANALYZE`. When set to any value, the feature will be turned on.
```

Comment:
```suggestion
: List of experimental features.  Features are neither initiated nor enabled by default. Turn on a  feature by adding it to the `[features]` .  Set the feature to `true`  to enable and `false` to disable. Features added to the list must be enabled or disabled. The above example initiates and enables one feature, which is the `analyze` feature. You can also turn on and enable or disable any feature flag from the command line using an environment variable.  For example, `CHEF_FEAT_ANALYZE`  is the environment variable for the `analyze` feature and setting `CHEF_FEAT_ANALYZE=true` enables the `analyze` feature.
```

Reply:
I'm not sure that I've caught the nuances correctly. Let me know.

Reply:
Looking now 👁]👁